src/rocketmq_client.erl

```diff
@@ -33,7 +33,7 @@
         , code_change/3
         ]).
 
--record(state, {requests, opaque_id, sock, servers, opts, last_bin = <<>>}).
+-record(state, {requests, opaque_id, sock, sock_send_mod = gen_tcp, servers, opts, last_bin = <<>>}).
```

Comment:
Maybe just sock_mod?

Reply:
Yeah, that is descriptive enough. Will change.

Reply:
Fixed